lucene/ivy-versions.properties

```diff
@@ -54,8 +54,8 @@ com.sun.jersey.version = 1.19
 /commons-cli/commons-cli = 1.4
 /commons-codec/commons-codec = 1.13
 /commons-collections/commons-collections = 3.2.2
-/commons-io/commons-io = 2.6
-# necessary to run test or embedded Zookeeper as of 3.6.1
+/commons-io/commons-io = 2.8.0
+# necessary to run test or embedded Zookeeper as of 3.common-build.xml6.1
```

Comment:
a mistake?

Reply:
thanks @dsmiley, indeed it was a mistake